Demo/Application/Features/VenmoViewController.swift

```diff
@@ -12,7 +12,7 @@ class VenmoViewController: PaymentButtonBaseViewController {
         super.heightConstraint = 150
         super.viewDidLoad()
         venmoClient = BTVenmoClient(
-            apiClient: apiClient,
+            authorization: DemoConstants.sandboxTokenizationKey,
```

Comment:
Should be able to use `authorization` from `PaymentButtonBaseViewController` here.